hub/graphql/dataloaders.py

```diff
@@ -21,14 +21,15 @@
 class BasicFieldDataLoader(dataloaders.BaseDjangoModelDataLoader):
     field: str
     filters: dict = {}
+    select_related: list = []
```

Comment:
Do these need to be = dict() and = list() for immutability's sake?

Reply:
I don't think changing them to `dict()` and `list()` would fix the immutability issue, but I think simply removing the default values works – the factory sets them to fresh `{}` and `[]` on class creation.